app/javascript/redux/greetings/greetings.js

```diff
@@ -0,0 +1,41 @@
+// const initState = '';
+
+// const greetingReducer = (state = initState, action) => {
+//   switch (action.type) {
+//     case 'GET_GREETING':
+//       return action.payload;
+//     default:
+//       return state;
+//   }
+// };
+
+
+// export default greetingReducer;
```

Comment:
- [ ] @mohamedgamaleldin999999 Great work on your 'Hello world' project as one app, I loved the way you executed it, however you forgot to remove the commented code above, its not good practise having commented code in your codebase, kindly consider removing it